src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest.java

```diff
@@ -636,6 +655,103 @@ public void testCanMovePrisonerFromPrisonToHospital() {
+            .containsExactly(
+                Tuple.tuple("TRANSFER", "FROMTOL", "SYI", "Offender admitted to SHREWSBURY for reason: Recall From Intermittent Custody from Court 1."),
+                Tuple.tuple("PRISON", "RELEASE", "SYI", "Released from SHREWSBURY for reason: Conditional Release (CJA91) -SH Term>1YR.")
+            );
+    }
+
+    @Test
+    public void testCanReleasePrisonerFromPrisonToHospitalInNomis() {
+        final var token = authTokenHelper.getToken(AuthToken.CREATE_BOOKING_USER);
+
+        final var body = Map.of(
+            "lastName", "FromNomis",
+            "firstName", "ReleasedToHospital",
+            "dateOfBirth", LocalDate.of(2000, 10, 17).format(DateTimeFormatter.ISO_LOCAL_DATE),
+            "gender", "M",
+            "ethnicity", "M1");
+
+        final var entity = createHttpEntity(token, body);
+
+        final var createResponse = testRestTemplate.exchange(
+            "/api/offenders",
+            POST,
+            entity,
+            new ParameterizedTypeReference<String>() {
+            }
+        );
+
+        final var offenderNo = new Gson().fromJson(createResponse.getBody(), Map.class).get("offenderNo");
+
+        final var newBookingBody = Map.of("prisonId", "SYI", "fromLocationId", "COURT1", "movementReasonCode", "24", "youthOffender", "true", "imprisonmentStatus", "CUR_ORA", "cellLocation", "SYI-A-1-1");
+        final var newBookingEntity = createHttpEntity(token, newBookingBody);
+
+        final var newBookingResponse = testRestTemplate.exchange(
+            "/api/offenders/{nomsId}/booking",
+            POST,
+            newBookingEntity,
+            new ParameterizedTypeReference<String>() {
+            },
+            offenderNo
+        );
+        assertThat(newBookingResponse.getStatusCodeValue()).isEqualTo(200);
+
+        final var bookingId = new BigDecimal(new Gson().fromJson(newBookingResponse.getBody(), Map.class).get("bookingId").toString()).toBigInteger().longValue();
+        final var releaseBody = createHttpEntity(token, Map.of("movementReasonCode", "HP", "commentText", "released prisoner to hospital in NOMIS"));
+
+        final var releaseResponse = testRestTemplate.exchange(
+            "/api/offenders/{nomsId}/release",
+            PUT,
+            releaseBody,
+            new ParameterizedTypeReference<String>() {
+            },
+            offenderNo
+        );
+
+        assertThat(releaseResponse.getStatusCodeValue()).isEqualTo(200);
+
+        final var lastMovement =  testRestTemplate.exchange(
+            "/api/bookings/{bookingId}/movement/{sequenceNumber}",
+            GET,
+            createHttpEntity(token, null),
+            new ParameterizedTypeReference<Movement>() {
+            },
+            bookingId, 2
+        );
+
+        assertThat(lastMovement.getBody().getFromAgency()).isEqualTo("SYI");
+        assertThat(lastMovement.getBody().getToAgency()).isEqualTo("OUT");
```

Comment:
The key part of this test is that it is not possible to enter a hospital if releasing in NOMIS so the movement's to location is out. Therefore we don't want the updated text on the case note as we don't know which hospital the offender was moved to.